clients/cobol-lsp-vscode-extension/src/services/LanguageClientService.ts

```diff
@@ -30,6 +31,8 @@ import {Middleware} from "./Middleware";
 export class LanguageClientService {
     private readonly jarPath: string;
 
+    public languangeClient: LanguageClient;
+
     constructor(private middleware: Middleware) {
```

Comment:
```suggestion
    constructor(private middleware: Middleware, public languangeClient: LanguageClient) {
```

Reply:
We can't do that because the client creates inside the `start` function. I agree, that we must do refactoring.